src/App.tsx

```diff
@@ -1,29 +1,55 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/indent */
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const TOTAL_ITEMS = 42;
+const items = getNumbers(1, TOTAL_ITEMS).map(n => `Item ${n}`);
+const itemsPerPage = [3, 5, 10, 20];
```

Comment:
move this CONSTANTS to a separate - `utils/constants` file
```suggestion
const TOTAL_ITEMS = 42;
const itemsPerPage = [3, 5, 10, 20];
```